odd-platform-api/src/test/java/org/opendatadiscovery/oddplatform/BaseIntegrationTest.java

```diff
@@ -0,0 +1,55 @@
+package org.opendatadiscovery.oddplatform;
+
+import java.util.concurrent.atomic.AtomicInteger;
+import lombok.SneakyThrows;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.util.TestPropertyValues;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import static org.testcontainers.containers.PostgreSQLContainer.POSTGRESQL_PORT;
+
+@ActiveProfiles("integration-test")
+@SpringBootTest
+@ContextConfiguration(initializers = {BaseIntegrationTest.Initializer.class})
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_CLASS)
+public abstract class BaseIntegrationTest {
+
+    static final PostgreSQLContainer<?> POSTGRE_SQL_CONTAINER;
+
+    static {
+        POSTGRE_SQL_CONTAINER = new PostgreSQLContainer<>(DockerImageName.parse("postgres:13.2-alpine"));
+        POSTGRE_SQL_CONTAINER.start();
+    }
+
+    static class Initializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
+
+        public void initialize(final ConfigurableApplicationContext configurableApplicationContext) {
```

Comment:
Do we need  this code? TC allows to specify test URL like this: 
`%test.quarkus.datasource.jdbc.driver=org.testcontainers.jdbc.ContainerDatabaseDriver`
`%test.quarkus.datasource.jdbc.url=jdbc:tc:postgresql:11:///ocr`
 So you don't care about host and credentials

Reply:
yes, we need it, because we create new database for each test class, thus can't define name in properties